conf/messages

```diff
@@ -362,11 +362,11 @@ error.range.day=Day out of range
 error.range.month=Month out of range
 error.range.year=Year out of range
 error.date.invalid=Invalid date entered
-error.invalid.date.format=Enter a valid date
-error.enter_a_date=Enter a date
-head.error.enter_a_date=You need to enter a date
+error.invalid.date.format=Date you started to live here must be a real date
+error.enter_a_date=Enter the date you started to live here
+head.error.enter_a_date=Enter the date you started to live here
```

Comment:
Looks like a duplicated key

Reply:
All the views are using the same component for the error summary on top of the page which adds the head. prefix to the error message, if I change it to not use that it will affect other messages and screens that were not being changed in this audit